docs/MAINTAINERS-guidelines.md

```diff
@@ -0,0 +1,57 @@
+# Introduction
+All Hyperledger projects MUST have a `MAINTAINERS` file (`MAINTAINERS.md` or `MAINTAINERS.rst`) at the top-level directory of the source code. This document will provide guidelines on what should be included in the `MAINTAINERS` file.
+
+# Guidelines
+## List of Project Maintainers
+The first thing that MUST be included in the `MAINTAINERS` file is a list of the project's maintainers, both active and emeritus.
+
+It is recommended that the lists be sorted alphabetically and contain the maintainers name, GitHub ID, LFID, Chat ID, Email, Company Affiliation, and Scope.
+
+**NOTES:**
+* There MUST be at least one reliable mechanism to contact the maintainer (either chat ID or email).
+* Scope is dependent on the project and may not exist for a given project. Scope could be the whole project, a specific repository, specific directories in a repository, or high-level description of responsibility (e.g., Documentation).
+
+The following shows the suggested format for the information:
```

Comment:
I would like to make sure that the maintainers fell free to choose the format they prefer especially if they plan to use GitHub hooks as David was suggesting during the TSC call.

Reply:
Agree. That is why it is a "suggested format"